contracts/truefi3/interface/IDebtInstrument.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC721} from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+/**
+ * @title IDebtInstrument
+ * @notice Represents debt in the TrueFi protocol
+ * @dev This is the basic interface for representing Debt in TrueFi
+ * This interface is meant to be extended to support different debt types
+ *
+ * An NFT represents some debt where:
+ *  - Each debt has an owner and a borrower
+ *  - The owner mints and funds a loan
+ *  - The borrower can withdraw funds from the loan
+ *  - The borrower calls repay() to pay back the loan
+ *  - Repayments are withdrawable by the owner
+ *  - Every DebtInstrument has an underlying token
+ *  - Only principal debt is tracked
```

Comment:
Using withdraw word for both borrower and owner is confusing. Should we replace first one with `borrow`?